src/app/store/effects/users.effects.ts

```diff
@@ -465,8 +465,8 @@ export class UsersEffects {
           switchMap(res => {
             return [new CreateDomainSuccess(res)];
           }),
-          catchError(err => [
-            new CreateDomainFailure(err.error)
+          catchError(errorResponse => [
+            new CreateDomainFailure(errorResponse.error.error.error)
```

Comment:
@Waseem826 doesn't it look strange to have `errorResponse.error.error.error `?

Reply:
Yes. `errorResponse.error` returns error of API request and last `error.error` is to get to actual error returned from backend.